src/main/js/components/table/tableNavigationWorker.js

```diff
@@ -101,7 +102,17 @@ export function getKeyboardShortcuts() {
       );
     },
     text: (event) => {
-      if (selectedCell && !selectedCellEditing
+      if (selectedCell && event.ctrlKey && event.key === "c"  // Cell copy
+        && selectedCell.kind !== "concat") {
+        event.preventDefault();
+        event.stopPropagation();
+        ActionCreator.copyCellContent(selectedCell);
+      } else if (selectedCell && event.ctrlKey && event.key === "v"  // Cell paste
```

Comment:
Like @alexvetter suggested, here another occurrence of `ctrlKey` and maybe missing `Cmd` key